product_docs/docs/biganimal/release/overview/deployment_options/postgres_admin_user.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: BigAnimal's cloud account ba_admin_user
```

Comment:
This page should talk about getting superuser for BYOA.

Reply:
Is the self service superuser feature specific to BYOA and the ba_admin_user specific to BAH?

Reply:
No; BAPG is for all BigAnimals, and self-service superuser is only for BYOA.